apps/editing-toolkit/editing-toolkit-plugin/whats-new/whats-new.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * WPCOM addition to Gutenberg post editor menu
+ *
+ * Enqueues JS modifications to add What's New option
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Class Whats_New
+ */
+class Whats_New {
+	/**
+	 * Class instance.
+	 *
+	 * @var Whats_New
+	 */
+	private static $instance = null;
+
+	/**
+	 * Whats_New constructor.
+	 */
+	public function __construct() {
+		add_action( 'enqueue_whats_new_assets', array( $this, 'enqueue_script_and_style' ), 100 );
```

Comment:
I think this doesn't quite work as you expect: `add_action` in this case is defining an action called "enqueue_whats_new_assets". But that action is never called anywhere. For an action to do anything, you would have to write `do_action( 'enqueue_whats_new_assets' )`.

Basically, the problem is that you actually just want this function (`enqueue_script_and_style`) to run whenever a _different_ action runs. That action is called `enqueue_block_editor_assets`, and Gutenberg calls `do_action( 'enqueue_block_editor_assets' )` when it is loading: so if you hook into that action, your function here will be run when Gutenberg tells it to.

It just needs to be changed to:
```suggestion
		add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_script_and_style' ), 100 );
```